syntax = "proto3";

//telemetry input message
message TelemetryInput {
    float cpu_used = 1;
    float memory_used =2;
    float net_used = 3;
    float time_taken =4;
    string device_name=5;
}


message TelemetryOutput {
    string response = 1;
}

service HuggingfaceAdapter {
    rpc telemetry(TelemetryInput) returns (TelemetryOutput) {}
}

